app/src/test/java/com/github/sdp_begreen/begreen/exceptions/DatabaseTimeoutExceptionTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.github.sdp_begreen.begreen.exceptions
+
+import org.hamcrest.CoreMatchers.*
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.junit.Assert.assertThrows
+
+@RunWith(JUnit4::class)
+class DatabaseTimeoutExceptionTest {
+
+    @Test
+    fun databaseTimeoutExceptionCorrectlyThrowException() {
+        assertThrows(DatabaseTimeoutException::class.java) {
+            throw DatabaseTimeoutException()
```

Comment:
this is not a particularly useful test :p

Reply:
I agree, it was mostly to test that the default constructor worked, but it's indeed not very useful 😅